commands/update-ui/update-ui.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+shopt -s inherit_errexit
+
+. "$(dirname "${BASH_SOURCE[0]}")/../utils.sh"
+. "$(dirname "${BASH_SOURCE[0]}")/../cmd_runner.sh"
+
+main() {
+  # setup.
+  cmd_runner_setup
+
+  rust_version="$1"
+  if [[ -z "${rust_version}" ]]; then
+    die "missing rust version argument"
+  fi
+
+  # This script uses rustup to install the required rust version.
+  # Doing that in CI feels rather ugly, but sadly there's currently no mechanism
+  # in command-bot to use a different CI image for individual jobs, so this is
```

Comment:
This sounds like a nice feature to have.  

@mordamax currently we're using [paritytech/ci-linux:production](https://github.com/paritytech/command-bot/blob/a23c5335e50e12a3f09cf4d5247a7a77ae5441e8/helm/values-parity-prod.yaml#L15), as far as I understand.  
But does anything stop us from allowing that to be configured in the command scripts?

Reply:
I thought it changes the machine to execute the job on, not the image...

Reply:
yea while i thought different machine would have different image? 
but yea, you're right, now i paid attention that we have this configured on the app level 
```cjs
process.env.GITLAB_JOB_IMAGE ??= "paritytech/ci-linux:production"
```
yea, technically abs possible to make it as part of .cmd.json, but would that be ok overall? i mean secure etc

cc @altaua @alvicsam

Reply:
Yeah, that'd be useful, and no particular security concern. (For example, the CI image for normal CI jobs is configured in the .gitlab-ci.yml of the repo being tested; being able to configure it here in `command-bot-scripts` is no worse than that.)

Reply:
yea i can add it then to a .cmd.json config

Reply:
https://github.com/paritytech/command-bot/issues/211